routes/api.php

```diff
@@ -49,3 +54,10 @@
         Route::delete('/{user}', DeleteUserController::class)
             ->whereNumber('user');
     });
+
+Route::post('employees', StoreEmployeesController::class)->name('employees');
+Route::get("employees", ListEmployeesController::class);
+
+Route::post("tasks", UpsertTaskController::class)->name('tasks');
+Route::get("tasks/{task}",FindTaskController::class);
+Route::get("tasks", ListTasksController::class);
```

Comment:
```suggestion
Route::post('employees', StoreEmployeesController::class)->name('employees.store');
Route::get("employees", ListEmployeesController::class)->name('employees.index');

Route::post("tasks", UpsertTaskController::class)->name('tasks.store');
Route::get("tasks/{task}",FindTaskController::class)->name('tasks.show');
Route::get("tasks", ListTasksController::class)->name('tasks.index');
```
You can refer to this [part of the Laravel docs](https://laravel.com/docs/12.x/controllers#shallow-nesting) to learn about what tends to be the standard for route names.

Reply:
Did this together with @tomisueldo suggestion of grouping them by the prefix employees and tasks